compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -2715,7 +2715,7 @@ private HashSet<String> getFieldNames(List<RecordLiteralNode.RecordField> specif
         return fieldNames;
     }
 
-    private String getKeyValueFieldName(BLangRecordKeyValueField field) {
+    public String getKeyValueFieldName(BLangRecordKeyValueField field) {
```

Comment:
Let's use default-access-level instead of making it public.
```suggestion
    String getKeyValueFieldName(BLangRecordKeyValueField field) {
```

Reply:
Address here [d97530c](https://github.com/ballerina-platform/ballerina-lang/pull/40621/commits/d97530cb3d589035d6d66759b831507e9b0852f7)